### PR TITLE
zvm/install_upi.md

### PR DESCRIPTION
**Pull request by dbenoit17**
~~Work in progress, please do not merge.~~

This doc is a derivative of https://github.com/openshift/installer/blob/master/docs/user/metal/install_upi.md.

### CODE REVIEW

File: docs/user/zvm/install_upi.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+## The pull secret that provides components in the cluster access to images for OpenShift components.
+pullSecret: ''
+## The default SSH key that will be programmed for `core` user.
+sshKey: ''
+```
+
+Create a directory that will be used by the OpenShift installer to provide all the assets. For example `test-bare-metal`,
+
+```console
+$ mkdir test-bare-metal
```
Comment (jichenjc) [inline docs/user/zvm/install_upi.md]: s/bare-metal/zvm/ ?

File: docs/user/zvm/install_upi.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+* Using a 3270 console, connect to each guest and clear the z/VM virtual reader
+```### 3270 console - for each z/VM guest
+logon <zvm_user> here
+cp query rdr all
+cp pur rdr all
+```
+
+* For each z/VM guest, punch the RHCOS kernel, parameters file, and initramfs into the guest's virtual reader
+  * example using `vmur` from another guest on the same z/VM hypervisor
```
Comment (jichenjc) [inline docs/user/zvm/install_upi.md]: so we need document the <another guest> as pre-requirements...

**Comment by crawford** [comment]
This is a docs change.

/override ci/prow/e2e-aws-fips
/override ci/prow/e2e-aws-scaleup-rhel7
/override ci/prow/e2e-aws-upgrade

**Pull request merged**